go-sdk/pkg/sdk/keptn.go

```diff
@@ -325,8 +377,14 @@ func (k *Keptn) createErrorFinishedEventForTriggeredEvent(event cloudevents.Even
 	commonEventData.Status = err.StatusType
 	commonEventData.Message = err.Message
 
-	finishedEventType, _ := keptnv2.ReplaceEventTypeKind(event.Type(), "finished")
-	keptnContext, _ := event.Context.GetExtension(KeptnContextCEExtension)
+	finishedEventType, err2 := keptnv2.ReplaceEventTypeKind(event.Type(), "finished")
+	if err2 != nil {
+		return *new(cloudevents.Event), fmt.Errorf("unable to create finished event: %v from %s", err2, event.Type())
```

Comment:
instead of creating an empty event, the method signature should shoule be changed to `*cloudenvetns.Event`. In a case of an error return `nil, err`